core-libs/setup/ssr/logger/loggers/express-server-logger.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { Request } from 'express';
+import { SsrOptimizationOptions } from '../../optimized-engine';
+
+export interface ExpressServerLoggerContext {
+  request: Request;
+  options?: Partial<SsrOptimizationOptions>;
+}
+
+export interface ExpressServerLogger {
+  log(message: string, context: ExpressServerLoggerContext): void;
+  warn(message: string, context: ExpressServerLoggerContext): void;
+  error(message: string, context: ExpressServerLoggerContext): void;
+  info(message: string, context: ExpressServerLoggerContext): void;
+  debug(message: string, context: ExpressServerLoggerContext): void;
+}
+
+/**
+ *
+ * Default logger used in SSR (ExpressJS) to enhance logs visible e.g. in monitoring tools e.g. Kibana.
+ * It logs It outputs a JSON with properties "message" and "context",
```

Comment:
[MINOR]

just typos
```suggestion
 * Default logger used in SSR (ExpressJS) to enhance logs visible e.g. in monitoring tools e.g. Kibana.
 * It outputs a JSON with properties "message" and "context",
```